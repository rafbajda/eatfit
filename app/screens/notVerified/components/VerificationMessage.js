import React from 'react';
import {
    VerificationMainMessageText,
    VerificationInfoMessageText
} from '../styles/verificationMessageStyles';
import { CenterContainer } from '../../../shared/styles/common';
import I18n from 'i18n-js';

const VerificationMessage = props => {
    const { user } = { ...props };
    const { t } = I18n;
    return (
        <CenterContainer>
            <VerificationMainMessageText>
                {t('info.accountIsNotVerified')}
            </VerificationMainMessageText>
            <VerificationInfoMessageText>
                {t('info.check')} {user && user.email ? user.email : 'email'}{' '}
                {t('info.andClickOnVerificationLink')}
            </VerificationInfoMessageText>
        </CenterContainer>
    );
};

export default VerificationMessage;
