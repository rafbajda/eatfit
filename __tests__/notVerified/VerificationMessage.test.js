import 'react-native';
import React from 'react';
import renderer from 'react-test-renderer';
import NavigationTestUtils from 'react-navigation/NavigationTestUtils';
import VerificationMessage from '../../app/screens/notVerified/components/VerificationMessage';

describe('VerificationMessage snapshot', () => {
    jest.useFakeTimers();
    beforeEach(() => {
        NavigationTestUtils.resetInternalState();
    });

    it('renders component', async () => {
        const tree = renderer.create(<VerificationMessage />).toJSON();
        expect(tree).toMatchSnapshot();
    });
});
