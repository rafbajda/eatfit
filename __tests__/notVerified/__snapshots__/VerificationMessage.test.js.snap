// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`VerificationMessage snapshot renders component 1`] = `
<View
  className="sc-gqjmRU hVdkzq"
  style={
    Object {
      "backgroundColor": "#fff",
      "flex": 1,
      "height": 1334,
    }
  }
>
  <Text
    className="sc-bdVaJa bAkMmE"
    style={
      Object {
        "color": "#000",
        "fontFamily": "System",
        "fontSize": 16,
      }
    }
    uppercase={false}
  >
    [missing "en.info.accountIsNotVerified" translation]
  </Text>
  <Text
    className="sc-bwzfXH jrEKTs"
    style={
      Object {
        "color": "#000",
        "fontFamily": "System",
        "fontSize": 16,
      }
    }
    uppercase={false}
  >
    [missing "en.info.check" translation]
     
    email
     
    [missing "en.info.andClickOnVerificationLink" translation]
  </Text>
</View>
`;
